cfgov/unprocessed/css/apps/owning-a-home/form-explainer.less

```diff
@@ -0,0 +1,447 @@
+            border-color: @gold-80;
+        }
+        &:focus,
+        &.has-attention {
+            border-color: @gold-80;
+            background: rgba( 255, 147, 27, 0.5 );
+        }
+        &:focus {
+            outline: 1px dotted @gold;
+            outline-offset: 1px;
+        }
+    }
+
+    .lt-ie9 .image-map_overlay__checklist {
+        background: none;
+        -ms-filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33FF931B,endColorstr=#33FF931B)";
+        filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33FF931B,endColorstr=#33FF931B)";
+        zoom: 1;
+        &:hover, &:focus, &.has-attention {
+            -ms-filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#7FFF931B,endColorstr=#7FFF931B)";
+            filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#7FFF931B,endColorstr=#7FFF931B)";
+        }
+    }
+
+    .image-map_overlay__definitions {
+        & {
+            background: rgba( 0, 94, 93, 0.2 );
+        }
+        &:hover,
+        &.hover-has-attention {
+            border-color: @teal;
+        }
+        &:focus,
+        &.has-attention {
+            border-color: @teal;
+            background: rgba( 0, 94, 93, 0.5 );
+        }
+        &:focus {
+            outline: 1px dotted @dark-teal;
+            outline-offset: 1px;
+        }
+    }
+
+    .lt-ie9 .image-map_overlay__definitions {
+        background: none;
+        -ms-filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33005E5D,endColorstr=#33005E5D)";
+        filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#33005E5D,endColorstr=#33005E5D)";
+        zoom: 1;
+        &:hover, &:focus, &.has-attention {
+            -ms-filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#7F005E5D,endColorstr=#7F005E5D)";
+            filter: ~"progid:DXImageTransform.Microsoft.gradient(startColorstr=#7F005E5D,endColorstr=#7F005E5D)";
+        }
+    }
+
+    .o-expandable-group .o-expandable__form-explainer {
+        display: none;
+        border-bottom: none;
+        margin: 0 0 unit( 3px / @base-font-size-px, em );
+        transition: transform 100ms ease-out;
+        transform: translateX( 0 );
+
+        &.o-expandable__expanded,
+        &.o-expandable__expanding,
+        &.has-attention {
+            transform: translateX( -( @grid_gutter-width / 2 ) ) !important;
+            .o-expandable_link {
+                color: @black;
+            }
+        }
+
+        &:hover {
+            .o-expandable_link {
+                color: @black;
+            }
+        }
+
+        .o-expandable_header {
+            z-index: 1000;
+        }
+
+        .o-expandable_header-left {
+            max-width: 90%;
+        }
+
+        &:last-child .o-expandable_content {
+            border-bottom: none;
+        }
+
+        .o-expandable_header {
+            padding: unit( ( @grid_gutter-width/2 )
```

Comment:
This is difficult to follow, could you break each `unit` function to it's own line to so it's clearer which value applies to which direction?

Reply:
Fixed.